tests/integration/mixer/util.go

```diff
@@ -188,7 +188,7 @@ func GetAndValidateAccessLog(ns namespace.Instance, t *testing.T, labelSelector,
 	}
 
 	retryFn := func(_ context.Context, i int) error {
-		content, err := shell.Execute(false, "kubectl logs -n %s -l %s -c %s ",
+		content, err := shell.Execute(false, "kubectl logs -n %s -l %s -c %s --tail=1000",
```

Comment:
This is restricting to get just 1000 logs.. but previously it should have gotten all the logs?

Reply:
```
 --tail=-1: Lines of recent log file to display. Defaults to -1 with no selector, showing all log lines otherwise
10, if a selector is provided.
```

should be `-1` though, I'll change that